shared/html/devtools-panel.html

```diff
@@ -36,8 +42,24 @@
       margin: 2px;
     }
 
+    .protection-button-on {
```

Comment:
<img width="696" alt="image" src="https://user-images.githubusercontent.com/338988/184617819-812660df-cb76-4843-ad1b-1e433a0f67fe.png">

I think we should make the borders consistent here, so specifying a radius and border.